src/domain/Monkey.java

```diff
@@ -0,0 +1,49 @@
+package domain;
+
+public class Monkey extends BananaEater {
+
+    private String kind;
+
+    public Monkey(String name, int weight, int height, String kind) {
+        this.name = name;
+        this.weight = weight;
+        this.height = height;
+        this.kind = kind;
+    }
+
+    public Monkey() {
+        this("Humanoid", 50, 120, "Primate");
+    }
+
+    public Monkey(String name) {
+        this(name, 50, 120, "Primate");
+    }
+    
+    
+
+    public void sleep() {
+        System.out.println("Primate is sleeping...");
+    }
+
+    @Override
+    public void EatABanana() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.